<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoLoadDialog</class>
 <widget class="QDialog" name="PhotoLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔바른고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>사진 불러오기</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>25</y>
     <width>180</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="title">
    <string>정보</string>
   </property>
   <widget class="QLabel" name="HeightInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="Width">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
     </font>
    </property>
    <property name="text">
     <string>가로 길이</string>
    </property>
   </widget>
   <widget class="QLabel" name="Height">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
     </font>
    </property>
    <property name="text">
     <string>세로 길이</string>
    </property>
   </widget>
   <widget class="QLabel" name="WidthInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileNameInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔바른고딕</family>
     </font>
    </property>
    <property name="text">
     <string>파일 이름</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="LoadImage">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>275</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="text">
    <string>사진 불러오기</string>
   </property>
  </widget>
  <widget class="QLabel" name="PhotoPreview">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveImage">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTipDuration">
    <number>-4</number>
   </property>
   <property name="text">
    <string>사진 저장</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LoadImage</sender>
   <signal>clicked()</signal>
   <receiver>PhotoLoadDialog</receiver>
   <slot>LoadPhoto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveImage</sender>
   <signal>clicked()</signal>
   <receiver>PhotoLoadDialog</receiver>
   <slot>SavePhoto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LoadPhoto()</slot>
  <slot>SavePhoto()</slot>
 </slots>
</ui>
